.gitignore

```diff
@@ -1,6 +1,4 @@
 # Ignoring directories.
-/.idea/
 /vendor/
 # Ignoring files.
-/composer.lock
```

Comment:
Not ignoring this file causes a little problem. I need to generate `composer.lock` file with the 5.3 PHP's version, otherwise Travis CI cannot perform the tests.